### PR TITLE
Add smoke tests for no extra_field and no training_wheels for keyless

### PR DESCRIPTION
**Pull request by heliuchuan**
## Description
This tests the keyless path without training wheels and when extra_field is empty.

## Type of Change
- [ ] New feature
- [ ] Bug fix
- [ ] Breaking change
- [ ] Performance improvement
- [ ] Refactoring
- [ ] Dependency update
- [ ] Documentation update

## Which Components or Systems Does This Change Impact?
- [ X] Validator Node
- [ X] Full Node (API, Indexer, etc.)
- [ ] Move/Aptos Virtual Machine
- [ ] Aptos Framework
- [ ] Aptos CLI/SDK
- [ ] Developer Infrastructure
- [ ] Other (specify)

## How Has This Been Tested?
This PR is just tests

## Checklist
- [ X] I have read and followed the [CONTRIBUTING](https://github.com/aptos-labs/aptos-core/blob/main/CONTRIBUTING.md) doc
- [ X] I have performed a self-review of my own code
- [ X] I have commented my code, particularly in hard-to-understand areas
- [X ] I identified and added all stakeholders and component owners affected by this change as reviewers
- [ X] I tested both happy and unhappy path of the functionality
- [X ] I have made corresponding changes to the documentation

### CODE REVIEW

File: testsuite/smoke-test/src/keyless.rs [hunk 1]
```diff
@@ -254,25 +298,77 @@ async fn get_transaction(
...
+}}
+"#,
+        KEYLESS_ACCOUNT_MODULE_NAME, KEYLESS_ACCOUNT_MODULE_NAME
+    );
+    let txn_summary = cli.run_script(root_idx, &jwk_patch_script).await.unwrap();
+    debug!("txn_summary={:?}", txn_summary);
+
+    let mut info = swarm.aptos_public_info();
+
+    // Increment sequence number since we patched a JWK
```
Comment (alinush) [inline testsuite/smoke-test/src/keyless.rs:360]: Do you wanna update this comment? There is no JWK update...

File: types/src/keyless/circuit_testcases.rs [hunk 2]
```diff
@@ -43,10 +43,13 @@ static SAMPLE_NONCE: Lazy<String> = Lazy::new(|| {
...
+
 /// TODO(keyless): Use a multiline format here, for diff-friendliness
 pub(crate) static SAMPLE_JWT_PAYLOAD_JSON: Lazy<String> = Lazy::new(|| {
     format!(
-        r#"{{"iss":"https://accounts.google.com","azp":"407408718192.apps.googleusercontent.com","aud":"407408718192.apps.googleusercontent.com","sub":"113990307082899718775","hd":"aptoslabs.com","email":"michael@aptoslabs.com","email_verified":true,"at_hash":"bxIESuI59IoZb5alCASqBg","name":"Michael Straka","picture":"https://lh3.googleusercontent.com/a/ACg8ocJvY4kVUBRtLxe1IqKWL5i7tBDJzFp9YuWVXMzwPpbs=s96-c","given_name":"Michael","family_name":"Straka","locale":"en","iat":1700255944,"exp":2700259544,"nonce":"{}"}}"#,
+        r#"{{"iss":"{}","azp":"407408718192.apps.googleusercontent.com","aud":"407408718192.apps.googleusercontent.com","sub":"113990307082899718775","hd":"aptoslabs.com","email":"michael@aptoslabs.com","email_verified":true,"at_hash":"bxIESuI59IoZb5alCASqBg","name":"Michael Straka","picture":"https://lh3.googleusercontent.com/a/ACg8ocJvY4kVUBRtLxe1IqKWL5i7tBDJzFp9YuWVXMzwPpbs=s96-c","given_name":"Michael","family_name":"Straka","locale":"en","iat":1700255944,"exp":2700259544,"nonce":"{}"}}"#,
```
Comment (alinush) [inline types/src/keyless/circuit_testcases.rs:51]: What changed here?

File: types/src/keyless/circuit_testcases.rs [hunk 3]
```diff
@@ -149,26 +152,21 @@ pub(crate) static SAMPLE_PK: Lazy<KeylessPublicKey> = Lazy::new(|| {
...
-                "18655740917490387600951832966394348192941354047055725417941402265405327519607",
-            ],
-        )
-        .unwrap(),
-        G1Bytes::new_unchecked(
-            "9962972005608211805886517403986821350415966400803168299718241440902533221411",
-            "20072668315945244829556525294962274935287178051981646789124308243130814901907",
-        )
-        .unwrap(),
+        G1Bytes::new_from_vec(hex::decode("bc1b31c0d35d8ea1086640f209dc1fda01d9b57e7aff138687549ece67f45780").unwrap()).unwrap(),
```
Comment (alinush) [inline types/src/keyless/circuit_testcases.rs:155]: Kewl!

File: types/src/keyless/circuit_testcases.rs (hunk 2 above)
Comment (heliuchuan) [reply types/src/keyless/circuit_testcases.rs:51]: I pulled the iss val into a constant

**Review by alinush — APPROVED** [review]
Beautiful! ❤️

File: testsuite/smoke-test/src/keyless.rs (hunk 1 above)
Comment (heliuchuan) [reply testsuite/smoke-test/src/keyless.rs:360]: Done

File: testsuite/smoke-test/src/keyless.rs [hunk 4]
```diff
@@ -254,25 +298,77 @@ async fn get_transaction(
...
+        swarm.chain_info().root_account().address(),
+    );
+    let jwk_patch_script = format!(
+        r#"
+script {{
+use aptos_framework::{};
+use aptos_framework::aptos_governance;
+use std::option;
+fun main(core_resources: &signer) {{
+    let framework_signer = aptos_governance::get_signer_testnet_only(core_resources, @0000000000000000000000000000000000000000000000000000000000000001);
```
Comment (zjma) [inline testsuite/smoke-test/src/keyless.rs:348]: nit: `@0x1`

File: types/src/keyless/circuit_testcases.rs (hunk 2 above)
Comment (zjma) [inline types/src/keyless/circuit_testcases.rs:51]: woah, from a real jwt?

**Review by zjma — APPROVED** [review]

File: types/src/keyless/circuit_testcases.rs (hunk 2 above)
Comment (heliuchuan) [reply types/src/keyless/circuit_testcases.rs:51]: michael is part of the protocol

**Pull request merged**